Add JS testing workshop (30.11.2019) (#53)
codecurious-bln/CC-website: en/_posts/2019-11-30-JS-testing-workshop.markdown

---
tags: intermediate js_testing_workshop
categories: en events
i18n_key: js_testing_workshop_2019_11
---

![Photo of JavaScript code on a laptop with cake on the side and an apple and lamp in the background.](/assets/images/JStesting-Nov19.jpg)

## Let’s dive deeper into Coding ♥️

You know how to code in JavaScript and want to learn testing with JS or improve your testing?

#### When: **30.11.2019, 09:30 - 16:00**
#### Where: **bitcrowd, Oranienstr 6, 10997 Berlin**

#### Apply:
The workshop is geared towards women* with coding experience in JavaScript. [**Apply Here**](https://forms.gle/dwyLkQXRsnVHGxnx6)

> Are a JavaScript coach? [**Apply here to coach at this workshop**](https://forms.gle/grVdR7y7hhZ9GCmu8).

Reviewer: I guess the “you” is missing here?

> Are a JavaScript coach? [**Apply here to coach at this workshop**](https://forms.gle/grVdR7y7hhZ9GCmu8).

Reviewer 2: ```suggestion
You are a JavaScript coach? [**Apply here to coach at this workshop**](https://forms.gle/grVdR7y7hhZ9GCmu8).
```

#### Registration Schedule

> - Registration starts: **13 November**

Reviewer: I don't know how it is usually handled. I noticed that the way you've written the date isn't perfectly clear. 
Maybe we could use something like “November 13th”? or “13. November”?

> - Registration starts: **13 November**

Reviewer 2: ```suggestion
- Registration starts: **13. November**
```

> - Registration ends: **28 November**

Reviewer 2: ```suggestion
- Registration ends: **28. November**
```

- Confirmation email sent: **There will be no confirmation email. If you sign up, we expect you at the workshop.**

## Workshop Schedule
### Saturday, 30 November - Workshop Day
- 09:30 - 10:00: **Coffee & Registration**
- 10:00 – 10:10: **Welcome Intro** by code curious
- 10:10 – 11:00: **Introduction** to JavaScript testing
- 11:00 - 11:15: **Break**
- 11:15 - 12:00: **Set-up for practical part** in groups

> - 12:00 – 13:00: **Lunch** (It will be provided by bitcrowd)

Reviewer 2: ```suggestion
- 12:00 – 13:00: **Lunch** (will be provided by bitcrowd)
```

- 13:00 - 13:20: **Lightning Talks**
- 13:20 – 16:00: **Write tests in groups**

## What to bring?
You will need a laptop (don't forget your charging cable). Feel free to come to the workshop at 09:00 am, if you have any problems with your computer.

## Venue

> We will meet in bitcrowd's office at Oranienstr 6, 10997 Berlin.

Reviewer 2: ```suggestion
We will meet in bitcrowd's event space at Oranienstr 6, 10997 Berlin.
```

<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2428.8534753300955!2d13.422867315900039!3d52.4998918798104!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47a84fac97094a8f%3A0x84c7ec5f6fb1f73!2sbitcrowd%20GmbH!5e0!3m2!1sen!2sde!4v1573605432282!5m2!1sen!2sde" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen=""></iframe>

### Accessibility
The venue is wheelchair accessible. Please write us an [E-Mail](mailto:contact@codecurious.org) in advance, so we can arrange the elevator key in time.

> Dogs are allowed on site, but please bring your dog only if they are very quiet and do not disturb everyone. Be considerate and respectful of other people's fears and write us an [email](mailto:contact@codecurious.org) in advance if plan on bringing your dog.

Reviewer 2: ```suggestion
Dogs are allowed on site, but please bring your dog only if they are very quiet and do not disturb people. Be considerate and respectful of other's fears and write us an [email](mailto:contact@codecurious.org) in advance if plan on bringing your dog.
```
